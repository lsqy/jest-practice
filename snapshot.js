export const generateConfig = () => {
    return {
        server: 'localhost',
        port: 8080,
        date: 2024
    }
}

export const generateAnotherConfig = () => {
    return {
        server: 'localhost',
        port: 8080,
        date: 2022
    }
}