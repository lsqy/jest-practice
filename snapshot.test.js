import { generateConfig, generateAnotherConfig } from './snapshot'

test('测试 generateConfig 函数', () => {
    expect(generateConfig()).toMatchSnapshot({
        date: 2024
    });
})

test('测试 generateAnotherConfig 函数', () => {
    expect(generateAnotherConfig()).toMatchSnapshot({
        date: 2022
    });
})