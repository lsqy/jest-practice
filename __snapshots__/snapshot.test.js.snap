// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 generateAnotherConfig 函数 1`] = `
Object {
  "date": 2022,
  "port": 8080,
  "server": "localhost",
}
`;

exports[`测试 generateConfig 函数 1`] = `
Object {
  "date": 2024,
  "port": 8080,
  "server": "localhost",
}
`;
